### PR TITLE
Add requirements.txt and README.md

### PR DESCRIPTION
**Pull request by SavagePastaMan**

### CODE REVIEW

**Comment by SavagePastaMan** [comment]
what should be in the readme?
- [ ] installation instructions
- [ ] description of the story
- [ ] how to play?

**Comment by SavagePastaMan** [comment]
is this ready to merge?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Flawless
+
+Hello :wave:, we are team AbstractSingletonProxyFactoryBean,
+and this is our submission for [PyWeek 31](https://pyweek.org/)
```
Comment (EwwPhysics) [inline README.md:4]: Add a period please

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+$ cd directory_name
+$ python -m pip install -r requirements.txt
+$ python -m game
+```
+Depending on your platform, you may need to use `python3` or `py`.
+
+### Background
+As part of the biggest heist of the century, your goal is to steal a diamond. To do this, you will have to bypass many levels of security. As you get closer to the diamond, the security will become tougher and you will face new challenges. 
+
+More information can be found by selecting `How to Play` from the main menu in the game.
```
Comment (EwwPhysics) [inline README.md:19]: Should we remove “in the game”?

Comment (GDWR) [reply README.md:19]: probs makes sense to remove it because it makes sense in context.

**Review by GDWR — CHANGES_REQUESTED** [review]
Not Enough Emojis 🙅

**Pull request merged**